cli/pkg/printer/printer.go

```diff
@@ -9,6 +10,7 @@ import (
 	"strings"
 
 	"github.com/fatih/color"
+	"github.com/gocarina/gocsv"
```

Comment:
What's wrong with the native `encoding/csv` package? It would be nice to avoid a new external dependency for this